test/mocha-utils.ts

```diff
@@ -277,3 +278,25 @@ export const mochaHooks = {
     sinon.restore();
   },
 };
+
+export const expectCookieEquals = (cookies, expectedCookies) => {
+  const { isChrome } = getTestState();
+  if (!isChrome) {
```

Comment:
I was talking about this one. 
If I understand the problem space right, we don't need it anymore.

Reply:
Oh you mean you don't want to assert the properties even in chrome? That's fine I just assumed we preferred to keep checking those on the browsers which supported them. Will update!

Reply:
To be clear, I need to reverse the logic in my current patch. 

If we want to assert only standard properties on all browsers, then I'll have to remove `sameParty`, `sourcePort` and `sourceScheme` from all the tests. And in the helper, instead of filtering out properties from `expectedCookies`, I will extract them from `cookies`. Because Chrome will keep providing cookie objects with extra information.